lib/routes/add_funds/deposit_to_btc_address_page.dart

```diff
@@ -181,8 +181,11 @@ class DepositToBTCAddressPageState extends State<DepositToBTCAddressPage> {
     final liquidity = accountModel.currency.format(
       connected ? accountModel.maxInboundLiquidity : Int64(0),
     );
+    final bool showFee = maxAllowedDeposit <= accountModel.maxInboundLiquidity;
 
-    if (connected && showMinFeeMessage) {
+    if (showFee) {
+      return texts.invoice_btc_address_channel_not_needed(minSats, maxSats);
```

Comment:
Perhaps rename this variable showFeeMessage and use it as
```suggestion
    if (!showFeeMessage) {
      return texts.invoice_btc_address_channel_not_needed(minSats, maxSats);
```